setup.cfg

```diff
@@ -1,5 +1,53 @@
+[metadata]
+name = astroid
+description = An abstract syntax tree for Python with inference support.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/PyCQA/astroid
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+license = LGPL-2.1
+license_files = LICENSE
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+keywords = static code analysis,python,abstract syntax tree
+project_urls =
+    "Bug tracker" = https://github.com/PyCQA/astroid/issues
+    "Discord server" = https://discord.gg/kFebW799
+    "Mailing list" = mailto:code-quality@python.org
+
+[options]
+packages = find_namespace:
```

Comment:
`find:` was the right directive. You just need to add `astroid/brain/__init__.py`

Reply:
``__init__.py`` is not mandatory since python 3.3 I think ?

Reply:
Not exactly sure what you mean. `setuptools` just looks for the `__init__.py` file to determine if it's a package that should be included.

Reply:
https://stackoverflow.com/questions/37139786/is-init-py-not-required-for-packages-in-python-3-3 Seeing the second answer you're right.

Reply:
The pylint bug that prevent pylint from analysing directory without ``__init__.py`` is bitting us there. I don't have the time to fix all of those legacy warnings.